bravado/requests_client.py

```diff
@@ -93,6 +93,31 @@ def __init__(self):
         self.session = requests.Session()
         self.authenticator = None
 
+    @staticmethod
+    def separate_params(request_params):
+        """Splits the passed in dict of request_params into two buckets.
+
+        - sanitized_params are valid kwargs for constructing a
+          requests.Request(..)
+        - misc_options are things like timeouts which can't be communicated
```

Comment:
Can we use the variable name used inside as `misc_options` as well (instead of `request_options`) to make things clearer.